### PR TITLE
SetState Progress Logging

### PR DESCRIPTION
**Pull request by tst-jkent**

### CODE REVIEW

File: earth_enterprise/src/fusion/autoingest/sysman/AssetVersionD.cpp [hunk 1]
```diff
@@ -230,11 +250,17 @@ void
...
+         ToString(state).c_str(), 
+         GetRef().c_str());
+  notify(NFY_VERBOSE, "Iterate through parents");
   for (std::vector<std::string>::const_iterator p = parents.begin();
        p != parents.end(); ++p) {
     AssetVersionD parent(*p);
     if (parent) {
+      notify(NFY_VERBOSE, "parent: %s exists", 
+             ToString(parent).c_str());
+      notify(NFY_VERBOSE, "Calling parent->HandleChildStateChange(%s)",
```
Comment (tst-basilhuffman) [inline earth_enterprise/src/fusion/autoingest/sysman/AssetVersionD.cpp]: Add a notify( ) in AssetVersionImpl::HandleChildStateChange( ) also

**Comment by tst-basilhuffman** [comment]
Don't name public branches this way. Should just be setstate-progress-logging

**Comment by tst-jkent** [comment]
Stephen had pointed that out to me yesterday. I changed the PR name, should I go ahead and change the branch name as well? @tst-basilhuffman

**Comment by tst-basilhuffman** [comment]
> Stephen had pointed that out to me yesterday. I changed the PR name, should I go ahead and change the branch name as well? @tst-basilhuffman

At this point, it may not really matter. @tst-lsavoie ?

File: earth_enterprise/src/fusion/autoingest/sysman/AssetVersionD.cpp [hunk 2]
```diff
@@ -300,6 +344,7 @@ void
 AssetVersionImplD::HandleChildStateChange(const std::string &) const
 {
   // NoOp in base since leaves don't need to do anything
+  notify(NFY_VERBOSE, "HandleChildStateChange: %s", GetRef().c_str());
 }
 
 void
```
Comment (tst-basilhuffman) [inline earth_enterprise/src/fusion/autoingest/sysman/AssetVersionD.cpp]: I apologise, I should have been more specific. I also meant to say you should differentiate

**Pull request closed without merging**